ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -377,7 +390,7 @@ wan_virtual_topologies:
       application_virtual_topologies:
         - application_profile: VIDEO
           path_groups:
-            - names: [MPLS, INET]
+            - names: [MPLS, INET, Equinix]
```

Comment:
```suggestion
              # The expectation here is that Equinix will be configured with priority 1/preferred,
              # since we are overriding the default_preference value (2/alternate).
            - names: [MPLS, INET, Equinix]
```